src/storage/src/sink/kafka.rs

```diff
@@ -120,6 +121,13 @@ where
             Antichain::new()
         }));
 
+        let internal_cmd_tx = Rc::clone(
+            storage_state
+                .internal_cmd_tx
+                .as_ref()
+                .expect("missing internal command sender"),
+        );
```

Comment:
maybe someday we can have a better api than raw access to the the channel, but for now its fine!